integration/integration_test.go

```diff
@@ -4543,14 +4562,20 @@ func testAuditOff(t *testing.T, suite *integrationTestSuite) {
 	var err error
 
 	// create a teleport instance with auth, proxy, and node
+	sessionsDir := t.TempDir()
 	makeConfig := func() (*testing.T, []string, []*helpers.InstanceSecrets, *servicecfg.Config) {
+		auditConfig, err := types.NewClusterAuditConfig(types.ClusterAuditConfigSpecV2{
+			AuditSessionsURI: sessionsDir,
```

Comment:
I'm surprised this works. Doesn't `AuditSessionsURI` have to be a URI? (`file:///home/log`)

Reply:
I guess the scheme defaults to `file` if it's unset: https://github.com/gravitational/teleport/blob/capnspacehook/seperate-audit-record-events/api/utils/uri.go#L35